pkg/app/piped/executor/ecs/ecs.go

```diff
@@ -92,6 +92,20 @@ func loadServiceDefinition(in *executor.Input, serviceDefinitionFile string, ds
 		return types.Service{}, false
 	}
 
+	if serviceDefinition.PropagateTags == "" {
+		serviceDefinition.PropagateTags = types.PropagateTagsService
+	}
```

Comment:
Sorry, I missed this part. I think we should always propagate the tags from the service to its tasks due to tags contains out builtin tags as well. So it's better to set the value to CreateServiceInput.PropagateTags directly in CreateService method. WDYT?

Reply:
I agree.
I think we should always propagate tags from the service to its tasks.
So, set the value directly to CreateServiceInput.PropagateTags in the CreateService method.